exporters/otlp/src/otlp_http_exporter_factory.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/otlp/otlp_http_exporter_factory.h"
+#include "opentelemetry/exporters/otlp/otlp_http_exporter.h"
+#include "opentelemetry/exporters/otlp/otlp_http_exporter_options.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> OtlpHttpExporterFactory::Create()
+{
+  OtlpHttpExporterOptions options;
+  return Create(options);
```

Comment:
Should we use `Create(OtlpHttpExporterOptions{})` to reduce copy here?

Reply:
Likewise, the Create() method with an option takes a `const &` parameter,
so there should be no copy.